third_party/gfxreconstruct/framework/decode/file_processor.h

```diff
@@ -250,6 +257,10 @@ class FileProcessor
     int64_t                             block_index_to_{ 0 };
     bool                                loading_trimmed_capture_state_;
 
+    // GOOGLE: [single-frame-looping] Record info obtained from capture file
+    int64_t  state_end_marker_file_offset_{ 0 };
+    uint64_t single_frame_{ 0 };
```

Comment:
my first impression of seeing this var is that it is a bool that indicates if single frame mode is enabled or not. maybe better with `single_frame_loop_index_`?